.github/workflows/rustgen.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
           git fetch upstream --tags
 
       - name: Install uv and setup uv caching
-        uses: astral-sh/setup-uv@v5
```

Comment:
rustgen should also specify the full version. This makes dependabot misbehave that is it switches to only update major versions.